edb/server/server.py

```diff
@@ -1466,7 +1499,7 @@ async def _start_servers(
                 if fut.result() is not None
             })
 
-        if not servers:
+        if not servers and not (admin and port):
             raise StartupError("could not create any listen sockets")
```

Comment:
Please add a short comment here.

Reply:
And can we have a separate PR specifically for this change?

Reply:
Created #4138.